.cirrus.yml

```diff
@@ -86,18 +86,20 @@ iox2_common_build_and_test_no_doc_tests_release_template: &IOX2_COMMON_BUILD_AND
 
 iox2_ffi_common_debug_template: &IOX2_FFI_COMMON_DEBUG
   ffi_script:
-    - cmake . -B target/ffi/build -DCMAKE_INSTALL_PREFIX=target/ffi/install -DEXAMPLES=ON -DCMAKE_BUILD_TYPE=Debug
+    - cmake . -B target/ffi/build -DCMAKE_INSTALL_PREFIX=target/ffi/install -DEXAMPLES=ON -DBUILD_TESTING=ON -DCMAKE_BUILD_TYPE=Debug
     - cmake --build target/ffi/build
     - cmake --install target/ffi/build
+    - target/ffi/build/tests/iceoryx2-c-tests
     - rm -rf target/ffi/build
     - cmake examples/c -B target/ffi/out-of-tree -DCMAKE_PREFIX_PATH=$( pwd )/target/ffi/install -DCMAKE_BUILD_TYPE=Debug
     - cmake --build target/ffi/out-of-tree
 
 iox2_ffi_common_release_template: &IOX2_FFI_COMMON_RELEASE
   ffi_script:
-    - cmake . -B target/ffi/build -DCMAKE_INSTALL_PREFIX=target/ffi/install -DEXAMPLES=ON -DCMAKE_BUILD_TYPE=Release
+    - cmake . -B target/ffi/build -DCMAKE_INSTALL_PREFIX=target/ffi/install -DEXAMPLES=ON -DBUILD_TESTING=ON -DCMAKE_BUILD_TYPE=Release
```

Comment:
I discover the good old inconsistencies from iceoryx1. `BUILD_TESTING` but `EXAMPLES`. 
Could you please add the `BUILD_` prefix to `EXAMPLES` so that this is consistent again.

We could make it like `BUILD_` for building something extra and `FEATURE_` to enable a specific feature.